packages/data/src/api/contractMinters/contractMinters.ts

```diff
@@ -0,0 +1,24 @@
+import { Network } from '@mintbase-js/sdk';
+import { fetchGraphQl } from '../../graphql/fetch';
+import { ParsedDataReturn } from '../../types';
+import { parseData } from '../../utils';
+import { contractMintersQuery } from './contractMinters.query';
+import { ContractMintersResults } from './contractMinters.types';
+
+
+export const contractMinters = async (
+  contractAddress: string | string[],
```

Comment:
Does `string[]` work? I would assume that it crashes the query. Also what is returned then, a union of minters? Are duplicates removed? This would be a lot less confusing if we dropped to possibility to query multiple smart contracts at once.